scripts/Enum/avgle_parameter.py

```diff
@@ -0,0 +1,15 @@
+from enum import Enum
```

Comment:
I think it can split to two different classes for Python files.

The `AvgleTimeType` and `AvgleSeachType` should be `AvgleTimeType.py` and `AvgleSeachType.py` files.

Reply:
Splitting enum to two file is ok.
But most file name is snake case, do you wnat change file name to camel case?

Reply:
To be consistency, I prefer these Python scripts should have united file name and class name.

After looking all Python scripts, we can do following stesps:

- Python file name is with snake case will not have class and it should include only function.
- Python file name is with camel case will have class and it includes only one class.
And the file name is same as class name.

What do you think :)?

Reply:
I also prefer scripts should be united file name and class name.

Maybe we can use camel case in next pull request.